routing/payment_session_test.go

```diff
@@ -8,6 +8,54 @@ import (
 	"github.com/lightningnetwork/lnd/routing/route"
 )
 
+func TestValidateCLTVLimit(t *testing.T) {
+	t.Parallel()
+
+	testCases := []struct {
+		cltvLimit      uint32
+		finalCltvDelta uint16
+		includePadding bool
+		expectError    bool
+	}{
+		{
+			cltvLimit:      uint32(103),
+			finalCltvDelta: uint16(100),
+			includePadding: true,
+			expectError:    true,
+		},
+		{
+			cltvLimit:      uint32(104),
+			finalCltvDelta: uint16(100),
+			includePadding: true,
+			expectError:    false,
+		},
+		{
+			cltvLimit:      uint32(100),
+			finalCltvDelta: uint16(100),
+			includePadding: false,
+			expectError:    true,
+		},
+		{
+			cltvLimit:      uint32(101),
+			finalCltvDelta: uint16(100),
+			includePadding: false,
+			expectError:    false,
+		},
+	}
+
+	for _, testCase := range testCases {
+		err := ValidateCLTVLimit(
+			testCase.cltvLimit, testCase.finalCltvDelta,
+			testCase.includePadding,
+		)
+		if testCase.expectError && err == nil {
```

Comment:
I think we want to use `require` wherever we can in tests now, can be changed to sth like,
```go
if expected error:
    require.Error(t, err, "...")
    continue

require.NoError(t, err, "...")
```

Reply:
using require